tests/pauli/dla/test_lie_closure.py

```diff
@@ -501,6 +503,17 @@ def test_lie_closure_heisenberg_generators_even(self):
         res = qml.pauli.lie_closure(generators)
         assert len(res) == 4 * ((2 ** (n - 2)) ** 2 - 1)
 
+    @pytest.mark.parametrize("n, res", [(3, 4), (4, 12)])
+    def test_lie_closure_heisenberg(self, n, res):
+        """Test the resulting DLA from Heisenberg model with summed generators"""
```

Comment:
If you have a theoretical scaling law for the expected DLA size, it might be nice to leave it in the docstring here :)

Reply:
I dont 🙈 but for these sizes I am fairly sure of the results